apps/rpc-server/src/constants.ts

```diff
@@ -1,11 +1,11 @@
 import * as path from "path";
+require("dotenv").config({ path: path.join(process.cwd(), "./.env.local") });
```

Comment:
can you remind me why you needed to add this?

Reply:
It's needed to start easily the rpc-server without defining the 3 variables every time/
Removing it for now, as it didn't want to work for me.